frontend/src/components/requisitionForm/RequisitionItemCard.tsx

```diff
@@ -94,7 +102,7 @@ const RequisitionItemCard: React.FC<Props> = (props) => (
         <Form.Item
```

Comment:
I feel like the Notes field for an individual item should be optional

Reply:
Changed